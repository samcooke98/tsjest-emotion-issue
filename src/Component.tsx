import * as React from "react";
import styled from "react-emotion";

const Container = styled.div({
  padding: 48,
  backgroundColor: "#08ed12"
});

const Text = styled.p({
  fontFamily: "Helvetica",
  fontSize: 24,
  fontWeight: "bold"
});

const Component = () => (
  <Container>
    <Text>Hello World</Text>
  </Container>
);

export default Component;
