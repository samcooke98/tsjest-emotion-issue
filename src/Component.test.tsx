
import * as React from 'react';

import { create} from 'react-test-renderer'
import { createSerializer } from 'jest-emotion'
import * as emotion from 'emotion'

expect.addSnapshotSerializer(createSerializer(emotion))

import Component from './Component';

describe("A basic test", () => { 
    it("Should match the snapshot", () => { 
        expect(create(<Component/>)).toMatchSnapshot())
    })
})