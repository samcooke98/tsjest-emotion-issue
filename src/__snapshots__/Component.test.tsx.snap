// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A basic test Should match the snapshot 1`] = `
.emotion-2 {
  padding: 48px;
  background-color: #08ed12;
}

.emotion-0 {
  font-family: Helvetica;
  font-size: 24px;
  font-weight: bold;
}

<div
  className="emotion-2 emotion-3"
>
  <p
    className="emotion-0 emotion-1"
  >
    Hello World
  </p>
</div>
`;
